apps/morph_check

```diff
@@ -193,7 +193,7 @@ if __name__ == '__main__':
     yaml_path = args.config
     SEPARATOR = '=' * 40
 
-    config = {'neuron_checks': [], 'options': []}
+    config = {'neuron_checks': [], 'options': {}}
```

Comment:
The way to test the apps is to put most of the code in (tested) modules, then the app is just a few trivial lines that call a single function and pass the command line arguments to it.